pkgs/applications/science/misc/rink/default.nix

```diff
@@ -10,12 +10,11 @@ rustPlatform.buildRustPackage rec {
     rev = "v${version}";
     sha256 = "0vl996y58a9b62d8sqrpfn2h8qkya7qbg5zqsmy7nxhph1vhbspj";
   };
-  cargoPatches = [ ./cargo-lock.patch ];
 
-  # Delete this on next update; see #79975 for details
-  legacyCargoFetcher = true;
+  # Upstreamed in https://github.com/tiffany352/rink-rs/pull/53
+  cargoPatches = [ ./cargo-lock.patch ];
```

Comment:
This one has been merged in https://github.com/tiffany352/rink-rs/pull/53, so we'll be able to drop this patch once the next release comes out.